packages/flutter/test/widgets/selectable_text_test.dart

```diff
@@ -5582,6 +5582,7 @@ void main() {
   });
 
   group('context menu', () {
+    // Regression test for https://github.com/flutter/flutter/issues/169001.
     testWidgets(
       'iOS uses the system context menu by default if supported',
```

Comment:
nit: should this change to `iOS does not use the system context menu by default on SelectableText if supported`

Reply:
Good catch!